### PR TITLE
Feature new footer Kotlinlang + WebHelp

### PR DESCRIPTION
**Pull request by JuliaVolkova**

### CODE REVIEW

File: static/css/com/layout/footer.scss [hunk 1]
```diff
@@ -3,103 +3,231 @@
...
+  .g-footer-layout {
+    min-width: 960px;
+    max-width: 960px;
+    width: 100%;
+    margin: auto;
+    display: flex;
+    justify-content: space-between;
+
+    @media (max-width: 960px) {
+      width: 100%;
```
Comment (zoobestik) [inline static/css/com/layout/footer.scss:35]: exists in base selector for this rule and somethin else, does it need?

File: static/css/com/layout/layout.scss [hunk 2]
```diff
@@ -11,11 +11,13 @@
 
 .global-layout {
     min-height: 100%;
-    margin-bottom: -$footer-height;
```
Comment (zoobestik) [inline static/css/com/layout/layout.scss:14]: Oh, i know thats ugly, try to open [security.html](https://kotlinlang.org/security.html) or other short page.
I think it's better to improve body size :(

Comment (JuliaVolkova) [reply static/css/com/layout/layout.scss:14]: Fixed it. Flex box forever ❤️

File: static/css/com/layout/footer.scss (hunk 1 above)
Comment (JuliaVolkova) [reply static/css/com/layout/footer.scss:35]: It's redundant, I've deleted it. Thanks!

**Pull request merged**